Lib/idlelib/help_about.py

```diff
@@ -94,6 +95,7 @@ def create_widgets(self):
                      f"{version[:version.rindex('.')]}/library/idle.html",
                      justify=LEFT, fg=self.fg, bg=self.bg)
         docs.grid(row=7, column=0, columnspan=2, sticky=W, padx=10, pady=0)
+        docs.bind("<Button-1>", webbrowser.open_new(docs.text))
```

Comment:
Use webbrowser.open.   A lambda expression is needed so that the open call is delayed until a user clicks.
```suggestion
        docs.bind("<Button-1>", lambda: webbrowser.open(docs.text))
```